Zend/zend_exceptions.c

```diff
@@ -680,24 +680,24 @@ ZEND_METHOD(Exception, __toString)
 		}
 
 		if ((Z_OBJCE_P(exception) == zend_ce_type_error || Z_OBJCE_P(exception) == zend_ce_argument_count_error) && strstr(ZSTR_VAL(message), ", called in ")) {
-			zend_string *real_message = zend_strpprintf(0, "%s and defined", ZSTR_VAL(message));
+			zend_string *real_message = zend_strpprintf_unchecked(0, "%S and defined", message);
```

Comment:
Why is the unchecked version needed? I don't remember exactly when they should be used.

Reply:
%Z/%S are not valid printf formatters and thus the compiler will error that the number of args is incorrect.